Triage the remaining fallbacks

lezcano opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #121312

Building off work from @amjames. There may be some missclassifications, feel free to flag them. 

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler @amjames @desertfire @chauhang

lezcano committed:
Triage the remaining fallbacks

[ghstack-poisoned]

peterbell10 commented on torch/_inductor/lowering.py:
```diff
@@ -2245,6 +2241,127 @@ def is_aligned(x):
+make_fallback(aten.mode)
+make_fallback(aten.median)
+make_fallback(aten.nanmedian)
+make_fallback(aten.searchsorted)
```
I think this is actually pretty trivial, at least for CUDA. `bucketize` is already implemented and in eager `bucketize` just calls `searchsorted`, so I think change `bucketize` lowering into `searchsorted` and make `bucketize` a decomp.

lezcano committed:
Update on "Triage the remaining fallbacks"


Building off work from amjames. There may be some missclassifications, feel free to flag them. 

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]

lezcano committed:
Update on "Triage the remaining fallbacks"


Building off work from amjames. There may be some missclassifications, feel free to flag them. 

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]

lezcano committed:
Update on "Triage the remaining fallbacks"


Building off work from amjames. There may be some missclassifications, feel free to flag them. 

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler amjames desertfire chauhang

[ghstack-poisoned]
